apps/web/src/pages/EmployeeProfile/components/WfaSection/Display.tsx

```diff
@@ -0,0 +1,120 @@
+import { useIntl } from "react-intl";
+
+import {
+  EmployeeProfileWfaFragment,
+  WfaInterest,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getWfaInterestFirstPerson,
+} from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Ul } from "@gc-digital-talent/ui";
+
+import ToggleForm from "~/components/ToggleForm/ToggleForm";
+import ExperienceCard from "~/components/ExperienceCard/ExperienceCard";
+import { formattedDate } from "~/utils/dateUtils";
+import processMessages from "~/messages/processMessages";
+
+import messages from "../../messages";
+
+interface DisplayProps {
+  user: EmployeeProfileWfaFragment;
+}
+
+const Display = ({ user }: DisplayProps) => {
+  const intl = useIntl();
+  const notProvided = intl.formatMessage(
+    commonMessages.missingOptionalInformation,
+  );
+  const experiences = unpackMaybes(user.currentSubstantiveExperiences);
+  const communities = unpackMaybes(user.employeeProfile?.communityInterests);
+
+  return (
+    <div className="flex flex-col gap-y-6">
+      <ToggleForm.FieldDisplay
+        label={intl.formatMessage(messages.wfaSituation)}
+      >
+        {user.employeeWFA?.wfaInterest?.value
+          ? intl.formatMessage(
+              getWfaInterestFirstPerson(user.employeeWFA.wfaInterest.value),
+            )
+          : notProvided}
+      </ToggleForm.FieldDisplay>
+      {user.employeeWFA?.wfaInterest?.value &&
+        user.employeeWFA.wfaInterest.value !== WfaInterest.NotApplicable && (
+          <>
+            <ToggleForm.FieldDisplay
+              label={intl.formatMessage(messages.currentSubstantivePosition)}
+            >
+              {experiences.length > 0 ? (
+                <div className="mt-3 flex flex-col gap-y-3">
+                  {experiences.map((exp) => (
+                    <ExperienceCard
+                      key={exp.id}
+                      experienceQuery={exp}
+                      showEdit={false}
+                    />
+                  ))}
+                </div>
+              ) : (
+                notProvided
+              )}
+            </ToggleForm.FieldDisplay>
```

Comment:
The warning well for non-CPA needs to be in display mode too

<img width="609" height="602" alt="image" src="https://github.com/user-attachments/assets/6b91a17d-b87e-432b-b162-65e10db241a0" />

Reply:
Added in [add CPA warning to display](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14672/commits/bedf27c9e7b038af9f54c699b682a4b6a4a9b63e)